source/storing-data/archive-versions.rst

```diff
@@ -0,0 +1,226 @@
+
+.. note::
+
+   V0 Archives do not capture provenance data. As a result, if a V0 artifact is 
+   an ancestor to a V1 (or greater) artifact, it is possible for the `action.yaml`
+   to list Artifact UUIDs which are not present in the `artifacts` directory.
+
+Version 2
+---------
+Released across QIIME 2 versions `2017.9 <https://github.com/qiime2/qiime2/releases/tag/2017.9.0>`_
+(`changelog <https://forum.qiime2.org/t/qiime-2-2017-9-release-is-now-live/1160>`__)
+and `2017.10 <https://github.com/qiime2/qiime2/releases/tag/2017.10.0>`_
+(`changelog <https://forum.qiime2.org/t/qiime-2-2017-10-release-is-now-live/1626>`__),
+the directory structure of this format is identical to v1,
+but the ``action.yaml`` file has changed.
+
+In `PR #333 <https://github.com/qiime2/qiime2/pull/333>`_, 
+the Version 2 ArchiveFormat adds an ``output-name`` key to the ``action`` section of ``action.yaml``
+(unless the action type is ``import``),
+assigning it the output name registered to the relevant action.
+Prior to this change, if one action returned multiple artifacts of the same :term:`Semantic Type`,
+it was not possible to differentiate between them using provenance alone.
+
+In `PR #348 <https://github.com/qiime2/qiime2/pull/348>`_,
+it adds provenance support for :term:`Pipelines <Pipeline>`,
+adding the ``alias-of`` key to the ``action`` section of ``action.yaml``.
+See description in :ref:`action-block` for details.
+
+Version 3
+---------
+
+Released in QIIME 2 version `2017.12 <https://github.com/qiime2/qiime2/releases/tag/2017.12.0>`_ (`changelog <https://forum.qiime2.org/t/qiime-2-2017-12-release-is-now-live/2308>`__),
+`PR #356 <https://github.com/qiime2/qiime2/pull/356>`_,
+the directory structure of this format is identical to v1 and v2.
+
+With this release, QIIME 2 Actions are able to take variadic arguments,
+allowing users to pass collections of Artifacts (``List`` s and ``Set`` s).
+A YAML representer has been added so that ``action.yaml`` can represent ``Set`` s of Artifact inputs.
+These will show up in ``action.yaml`` as custom ``!set`` tags.
+
+Version 4
+---------
+
+Released in QIIME 2 version `2018.4 <https://github.com/qiime2/qiime2/releases/tag/2018.4.0>`_ (`changelog <https://forum.qiime2.org/t/qiime-2-2018-4-release-is-now-live/3946>`__),
+`PR #387 <https://github.com/qiime2/qiime2/pull/387>`_,
+this format adds citations to the directory format,
+adds a ``transformers`` section to ``action.yaml``,
+and aligns the structure of ``environment:framework`` (also in ``action.yaml``)
+to match the structure of ``environment::plugins::<some_plugin>``.
+
+Whenever an Action is run, its registered citations are captured.
+When saved, they are written to a ``citations.bib`` file
+inside the Archive's ``provenance`` directory.
+Citations for all of the current Result's ancestors are stored in their respective <UUID> directories
+(e.g. ``/<root_UUID>/provenance/artifacts/<ancestor_UUID>/citations.bib``).
```

Comment:
A general note, you use the approach of "this file is in this directory (which is in another directory)" in many places in this doc: how about you just list the archive path, like you have done here, instead? I think it is much clearer.